codegen/smithy-aws-swift-codegen/src/test/kotlin/software/amazon/smithy/aws/swift/codegen/ec2query/Ec2QueryHttpResponseBindingErrorGeneratorTests.kt

```diff
@@ -101,11 +101,11 @@ class Ec2QueryHttpResponseBindingErrorGeneratorTests {
                 public var _retryable: Swift.Bool = false
                 public var _isThrottling: Swift.Bool = false
                 public var _type: ClientRuntime.ErrorType = .client
-                public var nested: AwsEc2ClientTypes.ComplexNestedErrorData?
+                public var nested: EC2ProtocolClientTypes.ComplexNestedErrorData?
                 public var topLevel: Swift.String?
             
                 public init (
-                    nested: AwsEc2ClientTypes.ComplexNestedErrorData? = nil,
+                    nested: EC2ProtocolClientTypes.ComplexNestedErrorData? = nil,
```

Comment:
Name of this type changed because it matches the `sdkId` in this test's Smithy def.